### PR TITLE
Branch

### PR DESCRIPTION
**Pull request by lungustefan**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://lungustefan.uk/

Hello, I want an NS record on my domain to add it on cloudflare to be able to use Agro tunel and change my DNS records by myself and not need to make an PR all the time.

### CODE REVIEW

File: domains/lungustefan.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        "owner": {
+           "username": "InterJadify",
+           "email": "",
+           "discord": "753179409682399332",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.ZerVkVuk5RE7iT8ymXVcKhUkihyAdAbufacrfxq1u7NKQ4Hd0_PbY5FYDAR5uuVd-sCe7fOztiSKV7f9q1OI4HNGDqQxiaWqzom_lQsrIMW-gWUDsBG4Vo8gsq0wrw2th797i-JfsXy2crWaeXlX7X1kROV5KZhGwlSusnUrMp3jlN-uUM3b4hxUyEs3grbh4BiZ9CWShm-kLUs572OBpMigXh3aO0DcUl-BdYdlpSEdRQe8zwylJbFnetxG3qBaMcciGwImixY6V4qaqFwH18isGjDknsHB2WvXM8ekMZmSxgMyRtv1VRSbC22grzMeBazgWoj3x65KU3Z8C7KNmw.w-d8n6WalLAvFcWHqlRkcw.W7GJTwpNRB9pUehhfBXVlkOR1knSNSjvJWeDViV6u2h6AyBdwyaAyILEQzC5ishw0f209VgdyaCqocC7nJp_ZjWjAfcyw0ICLiUXM37Sn3Q.xQOo4RjPMh6dBbWiF1Polw"
+        },
+    
+        "record": {
+            "NS1": "teagan.ns.cloudflare.com",
+            "NS2": "yevgen.ns.cloudflare.com"
```
Comment (CuteDog5695) [inline domains/lungustefan.json:11]:
```suggestion
            "NS": ["teagan.ns.cloudflare.com", "yevgen.ns.cloudflare.com"]
```

File: domains/interjadify.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Jadify-eu",
+           "email": "jadifyeu@gmail.com",
+           "discord": "753179409682399332"
+        },
+    
+        "record": {
+            "TXT": "will be changed"
```
Comment (CuteDog5695) [inline domains/interjadify.json:9]: This does nothing

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/interjadify.json (hunk 2 above)
Comment (jadifydotco) [reply domains/interjadify.json:9]: I cannot find the file with that text ( my browser crashes when opening the domains folder

**Comment by omsenjalia** [comment]
You have NS records. Reason for that?

**Comment by jadifydotco** [comment]
Hello, I eant NS records because i will probably handle 10/20 subdomains & I will also use some Zero Trust which only works when the domain is in my Cloudflare account (up to my knowledge) I will also sometimes download things from my PC ( Home IP ) but rarely.

**Comment by CuteDog5695** [comment]
Still waiting for Willy's response :p

**Pull request merged**